### DISCUSSION TITLE
from ._sync.gotrue_client import SyncGoTrueClient  is Right?

### DISCUSSION THREAD

**Discussion by Antony-Jia** [supabase/supabase#13031] [Questions]
from ._sync.gotrue_client import SyncGoTrueClient is right?
Or from ._sync.client import SyncGoTrueClient?
Because gotrue_client dose not have function set_auth in supabase-py.

**Comment by GaryAustin1** [comment]
If the python client is like all the other clients setAuth (set_auth) was dropped last year.

**Reply by Antony-Jia** [reply]
it's very kind of you for reply.
In new version, how I achieve the same functionality is like set_auth?
I want frontend and backend use the same jwt token in fastapi.
Each request in fastapi need new supabase client with jwt as context.

**Reply by GaryAustin1** [reply]
I see you have asked in the repository which is probably the correct place.   The alternative used in the js world is to set the authentication header with the jwt in createClient.  That may be the way to go, but I don't use Python and was just flagging that set_auth is likely gone.

**Reply by Antony-Jia** [reply]
Thank you for replay 👍

**Comment by Antony-Jia** [comment]
I think set_session that had the same effect.
